onnxruntime/python/tools/quantization/registry.py

```diff
@@ -57,6 +58,7 @@
     "AveragePool": QLinearPool,
     "Concat": QLinearConcat,
     "Softmax": QLinearSoftmax,
+    "Where": QLinearWhere,
```

Comment:
I think it'd better we don't register it by default. We only quantize when user requests to quantize it directly

Reply:
Never mind. There is check on the input in QDQWhere.